src/AutoRest.CSharp/Common/Output/Expressions/Statements/MethodBodyStatement.cs

```diff
@@ -10,9 +10,6 @@ namespace AutoRest.CSharp.Common.Output.Expressions.Statements
     [DebuggerDisplay("{GetDebuggerDisplay(),nq}")]
     internal record MethodBodyStatement
     {
```

Comment:
Maybe something is wrong on github, but I cannot find my previous comments on this and I do not know if this has been replied.

Why we remove this Empty property? I assume this could make the code more comprehensive than just new MethodBodyStatement() which reads also like an empty line.